lib/stdlib/doc/src/ets.xml

```diff
@@ -2294,13 +2294,16 @@ true</pre>
 
     <func>
       <name name="update_element" arity="3" clause_i="1" since=""/>
+      <name name="update_element" arity="4" clause_i="1" since=""/>
       <name name="update_element" arity="3" clause_i="2" since=""/>
+      <name name="update_element" arity="4" clause_i="2" since=""/>
```

Comment:
The since attributes should be `since="OTP @OTP-18870@"`

Reply:
I'm on it.

Reply:
Done in last commit.